### DISCUSSION TITLE
URL of certain packages doesn't seem to work

### DISCUSSION THREAD

**Discussion by Yashwanth131091** [Homebrew/discussions#3906] [Casks]
I am trying to install mumps using brew-->

~%brew install brewsci/num/brewsci-mumps

I get the following errors. I am a relative beginner to this shell environment in Mac. Any help would be appreciated. Thank you in advance.

"Error: brewsci-mumps: Failed to download resource "brewsci-metis"
Download failed: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz"

"==> Downloading http://mumps.enseeiht.fr/MUMPS_5.3.5.tar.gz
curl: (28) Failed to connect to mumps.enseeiht.fr port 80 after 75007 ms: Operation timed out
Warning: Problem : timeout. Will retry in 1 seconds. 3 retries left.
curl: (28) Failed to connect to mumps.enseeiht.fr port 80 after 75004 ms: Operation timed out
Warning: Problem : timeout. Will retry in 2 seconds. 2 retries left.
curl: (28) Failed to connect to mumps.enseeiht.fr port 80 after 75006 ms: Operation timed out
Warning: Problem : timeout. Will retry in 4 seconds. 1 retries left.
curl: (28) Failed to connect to mumps.enseeiht.fr port 80 after 75004 ms: Operation timed out

Error: brewsci-mumps: Failed to download resource "brewsci-mumps"
Download failed: http://mumps.enseeiht.fr/MUMPS_5.3.5.tar.gz"

**Comment by SMillerDev** [comment]
Try and contact brewsci or the mumps developers to try and debug this with you.

**Reply by Yashwanth131091** [reply]
thank you. I will contact the developers and post the results if any.
